qiskit/transpiler/passes/routing/commuting_2q_gate_routing/commuting_2q_gate_router.py

```diff
@@ -167,13 +167,11 @@ def run(self, dag: DAGCircuit) -> DAGCircuit:
         # Used to keep track of nodes that do not decompose using swap strategies.
         accumulator = new_dag.copy_empty_like()
 
-        self._bit_indices = {bit: index for index, bit in enumerate(dag.qubits)}
-
         for node in dag.topological_op_nodes():
             if isinstance(node.op, Commuting2qBlock):
 
                 # Check that the swap strategy creates enough connectivity for the node.
-                self._check_edges(node, swap_strategy)
+                self._check_edges(dag, node, swap_strategy)
```

Comment:
@mtreinish Just checking to see if passing the `dag` to helper functions here presents any problem.

Reply:
It shouldn't this function is only used internally by the pass and is explicitly marked as private so nothing should be depending on the function signature besides the internal pass code being updated here. So passing the dag like this to get the bits is fine. This actually fixes a potential state bug too, because before it was storing the bit indices as an instance variable.